### PR TITLE
VPN-5872: QML UI module cleanup

### PR DESCRIPTION
**Pull request by oskirby**
## Description
I am attempting to see if I can get the QML cache generation working in the `Mozilla.VPN` module, which should speed up first-boot load times significantly. To do this, we need to stop using `.qrc` files for loading QML content, and register them as QML sources in the `qt_add_qml_module()` CMake function instead.

The reason I stopped short of doing this last time we had to refactor the QML registration, amounted to two issues, which should now be dealt with:
- The generated resources for QML content was required to have a path matching the module URL. But we were mixing `qrc:/ui` and `import Mozilla.VPN` meaning that one or the other had to change.
- The QML type registration was getting lost during linking, and I think this has been resolved with the use of `Q_IMPORT_QML_MODULE(Mozilla_VPNPlugin)`

And finally, in order to remove the `ui.qrc` file altogether, we needed to dive into the translation string scraper job, and fix up [VPN-5515](https://mozilla-hub.atlassian.net/browse/VPN-5515) while we were at it.

## Reference
JIRA issue [VPN-5872](https://mozilla-hub.atlassian.net/browse/VPN-5872)
JIRA issue [VPN-5515](https://mozilla-hub.atlassian.net/browse/VPN-5515)
Qt blog: [QML Modules in Qt 6.2](https://www.qt.io/blog/qml-modules-in-qt-6.2)

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

[VPN-5872]: https://mozilla-hub.atlassian.net/browse/VPN-5872?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[VPN-5515]: https://mozilla-hub.atlassian.net/browse/VPN-5515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by strseb** [comment]
✨

**Review by lesleyjanenorton — APPROVED** [review]
👍

**Review by strseb — APPROVED** [review]
NICE!

File: scripts/utils/generate_ts.sh [hunk 1]
```diff
@@ -27,6 +27,17 @@ printn Y "Generating strings... "
 python3 cache/generate_strings.py src/translations/strings.yaml -o translations/generated
 print G "done."
 
+# Helper method - scan a directory and generate a QRC file which includes all QML files.
```
Comment (brizental) [inline scripts/utils/generate_ts.sh:30]: Note: CMake will actually generate generate this file if we just build the `mozillavpn-ui` module.

**Pull request merged**